games/n.yaml

```
@@ -149,6 +149,18 @@
   updated: 2015-05-09
   url: http://nstars.sourceforge.net/
 
+- name: NullpoMino
+  lang: Java
+  license:
+  - BSD
+  development: halted
+  originals:
+  - Tetris
+  repo: https://github.com/NullpoMino/NullpoMino
+  type: remake
+  updated: 2018-08-03
```

```suggestion
  updated: 2018-12-02
```
updated is w.r.t. this site